### PR TITLE
 Localization: Add Hindi translation

### PR DESCRIPTION
**Pull request by saurabhsharma2u**
#### Description
 Localization: Update Hindi (India) translation

Thank you!

### CODE REVIEW

File: src/localization/messages_Hi_IN.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+/*
+ * Translated default messages for the jQuery validation plugin.
+ * Locale: Hi-IN (Hindi; हिंदी)
```
Comment (bytestream) [inline src/localization/messages_Hi_IN.js:3]:
```suggestion
 * Locale: Hindi; हिंदी
```

**Review by bytestream — CHANGES_REQUESTED** [review]
I believe the file should be in ISO 639-1 format, so `message_hi.js`

File: src/localization/messages_Hi_IN.js (hunk 1 above)
Comment (saurabhsharma2u) [reply src/localization/messages_Hi_IN.js]: @bytestream Updated the suggested changes

**Review by bytestream — APPROVED** [review]

**Comment by saurabhsharma2u** [comment]
@bytestream thank you for review, please let know if anything else is required to merge request

**Pull request merged**